sympy/core/evalf.py

```diff
@@ -205,6 +229,8 @@ def complex_accuracy(result):
     In the worst case (re and im equal), this is wrong by a factor
     sqrt(2), or by log2(sqrt(2)) = 0.5 bit.
     """
+    if not isinstance(result, tuple):
+        return INF
```

Comment:
When would this be called with something that isn't a tuple? Maybe that should be fixed at the calling site rather than here.

Reply:
`zoo`, which isn't represented in mpmath. Originally `TEMP_RESULT` is not `Any` and `if not isinstance(result, tuple)` may make my type checker happy. I will change all of this to `if result is S.ComplexInfinity`.